website/src/categories/components/search-field/accessibility.md

```diff
@@ -6,8 +6,56 @@ eleventyNavigation:
   key: SearchFieldAccessibility
 ---
 
+
+<section>
+
+## Keyboard interactions
+
+Here's an overview of the common keyboard interactions associated with a search field:
+
+<div class="ds-table-wrapper">
+
+|Command|Description|
+|-|-|
+|Tab|When focus is outside the search field, moves focus to the input container. If focus is on the input container, moves focus to the end enhancer (if provided, and interactive).|
+|Esc|Claer search value, when is an input.|
+
+{.ds-table .ds-table-align-top}
+
+</div>
+
+</section>
+
+
+
 <section>
 
-TODO...
+## WAI-ARIA
+WAI-ARIA Roles, States, and Properties for a search field provide essential information to assistive technologies and screen readers. They convey the search field's role, state, and additional properties to ensure accessibility and a better user experience for individuals using assistive technology.
+<sl-tooltip id="tooltip1">Specifies whether the attribute is always set in the component (no) or it can be provided by the developer (yes)</sl-tooltip>
+
+<div class="ds-table-wrapper">
+
+|Attribute | Value | Description | User supplied <sl-icon name="info" aria-describedby="tooltip1" size="md"></sl-icon> |
+|-|-|-|-|
+|`role`|`'textbox'`|Identifies as a text input|no|
```

Comment:
This isn't correct. The search field itself wraps an `<input type="text">` and doesn't have a role.